shell/browser/api/atom_api_power_save_blocker.cc

```diff
@@ -90,7 +90,7 @@ device::mojom::WakeLock* PowerSaveBlocker::GetWakeLock() {
     mojo::Remote<device::mojom::WakeLockProvider> wake_lock_provider;
     DCHECK(content::GetSystemConnector());
     content::GetSystemConnector()->Connect(
-        device::mojom::kServiceName,
+        electron::kDeviceServiceName,
```

Comment:
@jkleinsc what happened to this constant? Why can't we use it any more? Is this service going away?

Reply:
https://chromium.googlesource.com/chromium/src/+/6071e000a31355d0eaca53d2e89f717c880b78e0%5E%21/#F16

Reply:
Looks like it was removed as its unused

Reply:
But I think the service is still around?
https://chromium.googlesource.com/chromium/src/+log/HEAD/content/browser/device/device_service.cc

Reply:
Hmm... maybe this was the reason: https://chromium.googlesource.com/chromium/src/+/45d15c28e4b5e66eeafb0f913f30d9d88e8e19e7

Reply:
We should move to use that interface as well `content::GetDeviceService()::BindWakeLockProvider()`